applications/dashboard/controllers/class.entrycontroller.php

```diff
@@ -1129,8 +1129,8 @@ public function signIn($method = false, $arg1 = false) {
                                 'Reason' => 'Password',
                             ]);
                         }
-                    } catch (Gdn_CoreException $ex) {
-                        $errorMessage = htmlspecialchars(strip_tags($ex->getMessage()));
+                    } catch (Gdn_SanitizedUserException $ex) {
+                        $errorMessage = htmlspecialchars($ex->getMessage());
                         $errorMessage .= sprintf(' Click <a href=%s>here</a> to reset your password', url('/entry/passwordrequest'));
```

Comment:
You should leave the link back in it's original spot now. The idea of the `SanitizedUserException` would be that the message is able to be used directly and we can leave the message where it originally came from.

Reply:
even without htmlspecialchars?